homeassistant/components/tado/sensor.py

```diff
@@ -272,14 +296,23 @@ def _async_update_home_data(self):
         try:
             tado_weather_data = self._tado.data["weather"]
             tado_geofence_data = self._tado.data["geofence"]
+            tado_aircomfort_data = self._tado.data["aircomfort"]
         except KeyError:
             return
 
         if self.entity_description.data_category is not None:
             if self.entity_description.data_category == SENSOR_DATA_CATEGORY_WEATHER:
                 tado_sensor_data = tado_weather_data
-            else:
+            elif self.entity_description.data_category == SENSOR_DATA_CATEGORY_GEOFENCE:
                 tado_sensor_data = tado_geofence_data
+            elif (
+                self.entity_description.data_category == SENSOR_DATA_CATEGORY_AIRCOMFORT
+            ):
+                tado_sensor_data = tado_aircomfort_data
+            else:
+                _LOGGER.error(
+                    "Unknown data category: %s", self.entity_description.data_category
+                )
```

Comment:
```suggestion
            else:  # SENSOR_DATA_CATEGORY_AIRCOMFORT
                tado_sensor_data = tado_aircomfort_data
```
I mean we have the sensor definition static here, there is no case where someone accidentally added one of these